### PR TITLE
fix(copy-paste): copy process ref instead of reusing

### PR DESCRIPTION
**Pull request by philippfromme**
Related to https://github.com/camunda/camunda-modeler/issues/3085.

### CODE REVIEW

File: test/spec/features/modeling/behavior/CreateParticipantBehaviorSpec.js [hunk 1]
```diff
@@ -536,4 +540,40 @@ describe('features/modeling - create participant', function() {
...
+          y: 1000
+        }
+      });
+
+      // then
+      var newParticipant = newElements[ 0 ];
+
+      expect(is(newParticipant, 'bpmn:Participant')).to.be.true;
+
+      expect(getBusinessObject(newParticipant).get('processRef')).not.to.equal(getBusinessObject(participant).get('processRef'));
```
Comment (nikku) [inline test/spec/features/modeling/behavior/CreateParticipantBehaviorSpec.js:574]: Should we assert what is being copied here? (Children, extension elements).

Or do we assert this in a different place (i.e. in `modelUtil` specs).

**Comment by philippfromme** [comment]
Okay, so it turns out we _are_ actually copying the `processRef` (cf. https://github.com/bpmn-io/bpmn-js/blob/develop/lib/features/copy-paste/BpmnCopyPaste.js#L67). For some reason, when I change the process ID and then copy and paste a participant the ID gets duplicated (cf. https://github.com/camunda/camunda-modeler/issues/3085).

**Comment by philippfromme** [comment]
I'll close this PR as this is not the proper fix. We'll need a fix for claiming IDs assigned to model elements.

**Comment by nikku** [comment]
I believe in the case we've discussed the `processRef` is simply not properly serialized with the model, as it is a _hidden_ reference. `JSON.stringify(...)` does not include these.

**Pull request closed without merging**